R/REST_helpers.R

```diff
@@ -36,12 +36,12 @@ sbtools_POST <- function(url, body, session, timeout = default_timeout(), ...){
 #' @keywords internal
 sbtools_GET <- function(url, ..., session, timeout = default_timeout()){
 	
-	supported_types <- c('text/plain','application/json')
+	supported_types <- c('text/plain','application/json','application/x-gzip')
```

Comment:
This does get tricky though, as someone could want to use sbtools to fetch an html doc that they have sitting up on sciencebase. I think this code would incorrectly recognize that action as a failed login. 

maybe a better way is to check that the response url != the login url...
note:
```R
r$url
[1] "https://my.usgs.gov/josso/signon/login.do?josso_back_to=https%3A%2F%2Fwww.sciencebase.gov%2Fcatalog%2Fjosso_security_check"
```
for a failed POST (due to auth)
vs a good one:
```R
r$url
[1] "https://www.sciencebase.gov/catalog/file/...
```